query/src/test/scala/filodb/query/exec/BinaryJoinGroupingSpec.scala

```diff
@@ -308,12 +309,12 @@ class BinaryJoinGroupingSpec extends AnyFunSpec with Matchers with ScalaFutures
       )
     )
     result.size shouldEqual 4
-    result.map(_.key.labelValues) sameElements(expectedLabels) shouldEqual true
+    result.map(_.key.labelValues).toSet shouldEqual expectedLabels.toSet
 
-    result(0).rows.map(_.getDouble(1)).toList shouldEqual List(0.75)
```

Comment:
Rows ordering has changed. Won't expectedLabels also change?

Reply:
RV ordering has changed. RV rows should remain same. ExpectedLabels should remain same as well.
These changes only account for RV order change